ayushi/ayushi_postgresql_assignment/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "ayushi_postgresql_assignment",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.ts",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
Can you please add script to build project.

Ref : any other folder which is merged